### ISSUE TITLE
Mutations add or remove cells even with zero propability for that.

### ISSUE THREAD

**Issue by somebodyy7** [MaxRobinsonTheGreat/LifeEngine#32]
**Describe the bug**
When both Add Cell and Remove Cell are set to 0, both of these types of mutation still seem to occur.
When only Add Cell is set to 0, organisms with added cells do appear.
I did not experience organisms with clearly removed cells if only Remove Cells is set to 0.

_I was not able to test if there is a similar bug for Change Cell, as the effect of every change-mutation could be achieved with removing and adding one cell._
_Using Global mutation rate set to 0 does not result in a similar bug._

**To Reproduce**
Before Mutation could occour, under Mutation Type Probabilities, set Add Cell or Remove Cell to 0.

_I restarted the enviroment after, to make sure all life would be affected the same.
I did not take these results from the Organism Structure statistic, as it seemed to be flawed.
I instead looked at organisms by selecting them into the Editor window._

**Expected behavior**
Mutations should not be able to add or remove cells.

**Environment:**
 - Desktop
 - Windows
 - Firefox

**Comment by MaxRobinsonTheGreat** [comment]
I am unable to reproduce this bug. If I set change cell to 100%, which forces the other two to 0%, it behaves as expected and no cells are added or removed. I believe you are probably having an issue with the ui, which automatically adjusts the values in a somewhat unintuitive way. This is a separate issue I plan to address.

**Comment by somebodyy7** [comment]
I did set Change Cell to 100%. I just repeated it like that two times each in Firefox, Microsoft Edge, and Pale Moon (freshly installed), making sure Simulation Controls were reset, and each time, after 1,800~10,000 ticks (mostly 5,000~6,000) and zooming and dragging, I found an organism with four cells instead of three (I selected it into the editor to make sure it wasn't two organisms overlapping).

**Comment by MaxRobinsonTheGreat** [comment]
Ah, I was able to reproduce by running it for a while and manually turning up the mutation rate. Turns out 0% was actually more like 1%, allowing these mutations to happen occasionally. Coincidentally it has been fixed by #43, which is in the develop branch and will be pushed soon. 

I've run the new code for ~10 minutes at full speed with add/remove cell set to 0, and no new cells are added or removed. Should be fixed.
